packages/carbon-graphs/src/js/controls/Scatter/Scatter.js

```diff
@@ -233,10 +266,7 @@ class Scatter extends GraphContent {
       .remove();
 
     updateShapesDuringReflow(graph, graphData, this);
-    this.valuesRange = calculateValuesRange(
-      this.config.values,
-      this.config.yAxis,
-    );
+    this.valuesRange[this.config.yAxis] = calculateValuesRangeYAxis(this.config.values, this.config.yAxis);
```

Comment:
The second parameter is no longer used (I missed this in the previous PRs, will correct them soon).
```suggestion
    this.valuesRange[this.config.yAxis] = calculateValuesRangeYAxis(this.config.values);
```

Reply:
Removed 49c8fce